pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -372,14 +373,6 @@ rec {
     inherit src version;
 
     patches = [
-      (fetchpatch2 {
-        name = "ceph-s3select-arrow-18-compat.patch";
-        url = "https://github.com/ceph/s3select/commit/f333ec82e6e8a3f7eb9ba1041d1442b2c7cd0f05.patch";
-        hash = "sha256-21fi5tMIs/JmuhwPYMWtampv/aqAe+EoPAXZLJlOvgo=";
-        stripLen = 1;
-        extraPrefix = "src/s3select/";
-      })
-
```

Comment:
Is that one known to not be needed anymore, or is that an educated guess based on "it builds"?
Either is fine by me, just asking out of curiosity.

Again, I don't have *s3select* in use so I couldn't run any meaningful tests anyway.

Reply:
The build detected this patch as already applied and suggested to revert.

I now did the detailed check:

* Ceph 19.2.3 [contains](https://github.com/ceph/ceph/tree/v19.2.3/src) submodule [s3select @ 0a0f6d4](https://github.com/ceph/s3select/tree/0a0f6d439441f5b121ed1052dac54542e4f1d89b)
* The [removed patch's commit](https://github.com/ceph/s3select/commit/f333ec82e6e8a3f7eb9ba1041d1442b2c7cd0f05) is [a parent](https://github.com/ceph/s3select/compare/f333ec82e6e8a3f7eb9ba1041d1442b2c7cd0f05...0a0f6d439441f5b121ed1052dac54542e4f1d89b) of that.